adapters/mqtt-vertx/src/main/java/org/eclipse/hono/adapter/mqtt/VertxBasedMqttProtocolAdapter.java

```diff
@@ -238,7 +240,7 @@ void handleEndpointConnection(final MqttEndpoint endpoint) {
 
     private void handleEndpointConnectionWithoutAuthentication(final MqttEndpoint endpoint) {
         endpoint.closeHandler(v -> {
-            LOG.debug("connection closed with client [{}]", endpoint.clientIdentifier());
+            LOG.info("connection closed with client [{}]", endpoint.clientIdentifier());
```

Comment:
This would log one line for each device going offline, right?
Maybe this would be too much noise in bigger setups?
I thought this was the reason to have it on level debug before. WDYT?